Grbl_Esp32/src/Probe.cpp

```diff
@@ -24,47 +24,40 @@
 #include "Grbl.h"
 
 // Inverts the probe pin state depending on user settings and probing cycle mode.
-uint8_t probe_invert_mask;
+bool is_probe_away;
 
 // Probe pin initialization routine.
 void probe_init() {
-#ifdef PROBE_PIN
-#    ifdef DISABLE_PROBE_PIN_PULL_UP
-    pinMode(PROBE_PIN, INPUT);
-#    else
-    pinMode(PROBE_PIN, INPUT_PULLUP);  // Enable internal pull-up resistors. Normal high operation.
-#    endif
-    probe_configure_invert_mask(false);  // Initialize invert mask.
+    static bool show_init_msg = true;  // used to show message only once.
+
+    if (PROBE_PIN != UNDEFINED_PIN) {
+#ifdef DISABLE_PROBE_PIN_PULL_UP
+        pinMode(PROBE_PIN, INPUT);
+#else
+        pinMode(PROBE_PIN, INPUT_PULLUP);  // Enable internal pull-up resistors. Normal high operation.
 #endif
```

Comment:
Future improvement:  Change Config.h to
`#define PROBE_PIN_MODE INPUT_PULLUP  // or INPUT`
Then this collapses to one line:
`pinMode(PROBE_PIN, PROBE_PIN_MODE);`
Similarly with DISABLE_LIMIT_PIN_PULL_UP and DISABLE_CONTROL_PIN_PULL_UP - although the control pin mode stuff is currently not implemented.